plugins/reporters/freemarker/src/main/kotlin/PlainTextTemplateReporter.kt

```diff
@@ -23,23 +23,40 @@ import java.io.File
 
 import org.ossreviewtoolkit.model.config.PluginConfiguration
 import org.ossreviewtoolkit.plugins.api.OrtPlugin
+import org.ossreviewtoolkit.plugins.api.OrtPluginOption
 import org.ossreviewtoolkit.plugins.api.PluginDescriptor
-import org.ossreviewtoolkit.plugins.reporters.freemarker.FreemarkerTemplateProcessor.Companion.OPTION_TEMPLATE_ID
-import org.ossreviewtoolkit.plugins.reporters.freemarker.FreemarkerTemplateProcessor.Companion.OPTION_TEMPLATE_PATH
 import org.ossreviewtoolkit.reporter.Reporter
 import org.ossreviewtoolkit.reporter.ReporterFactory
 import org.ossreviewtoolkit.reporter.ReporterInput
 
+data class PlainTextTemplateReporterConfig(
+    /**
+     * A comma-separated list of IDs of templates provided by ORT. Currently, only the "NOTICE_DEFAULT" and
+     * "NOTICE_SUMMARY" templates are available.
+     * If no template id or path is provided, the "NOTICE_DEFAULT" template is used.
+     */
+    @OrtPluginOption(aliases = ["templateId"])
+    val templateIds: List<String>?,
+
+    /**
+     * A comma-separated list of paths to template files provided by the user.
+     */
+    @OrtPluginOption(aliases = ["templatePath"])
+    val templatePaths: List<String>?
+) {
+    companion object {
+        internal val DEFAULT = PlainTextTemplateReporterConfig(
+            templateIds = listOf("NOTICE_DEFAULT"),
+            templatePaths = null
+        )
```

Comment:
How about moving this to the top level instead of wrapping it in a companion object?

Reply:
I actually like scoping such constants, to me `PlainTextTemplateReporterConfig.DEFAULT` also reads much better than `DEFAULT_PLAIN_TEXT_TEMPLATE_REPORTER_CONFIG`.

Reply:
The discussion we once had in the core team came to the conclusion that such scoping is only useful for public properties. Or maybe it was non-private properties? Then you would be right for the `internal` case. Let's keep it as-is then.